api/tests/unit/domain/usecases/find-sessions_test.js

```diff
@@ -0,0 +1,49 @@
+const { expect, domainBuilder } = require('../../../test-helper');
+const usecases = require('../../../../lib/domain/usecases');
+const { ForbiddenAccess } = require('../../../../lib/domain/errors');
+
+const User = require('../../../../lib/domain/models/User');
+
+describe('Unit | UseCase | find-sessions', () => {
+
+  it('should return sessions of the certificationCenter', async () => {
+    // given
+    const user = domainBuilder.buildUser();
+    const certificationCenterId = user.certificationCenterMemberships[0].certificationCenter.id;
+    const sessions = [domainBuilder.buildSession({ certificationCenterId })];
+    const sessionRepository = {
+      findByCertificationCenter: () => Promise.resolve(sessions)
+    };
+    const userRepository = {
+      getWithCertificationCenterMemberships: () => Promise.resolve(user)
+    };
+
+    // when
+    const sessionsFound = await usecases.findSessions({ userId: user.id, certificationCenterId, userRepository, sessionRepository });
+
+    // then
+    return expect(sessionsFound).to.be.deep.equal(sessions);
+  });
+
+  it('should throw an forbidden error if user is not a certification center member', () => {
```

Comment:
`should throw a forbidden error if user is not a member of the given certification center`